### ISSUE TITLE
.webm files crash the utility

### ISSUE THREAD

**Issue by anthonyw12123** [anthonyw12123/4chanwebscraper#10]
When processing a 4can thread, if it encounters a .webm file, it crashes immediately, saying;

```
saving:./wg - Official Homescreen Thread #88 - WallpapersGeneral - 4chan/1473537560676.webm
Traceback (most recent call last):
  File "4ChanWebScraper.py", line 39, in <module>
    i = Image.open(StringIO(img.content))
  File "/usr/local/lib/python2.7/site-packages/PIL/Image.py", line 2317, in open
    % (filename if filename else fp))
IOError: cannot identify image file <StringIO.StringIO instance at 0x1121a3290>
```

**Comment by anthonyw12123** [comment]
fix is in 1.1.1

**Comment by anthonyw12123** [comment]
Merged into 1.1.
